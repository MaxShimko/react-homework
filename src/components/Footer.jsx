import React from 'react';

import '../styles/Footer.scss';

export default function Footer() {
    return (
        <div className="footer">
            <div className="delimiter-2" />
            <div>
                <span className="logotype">
                    <span className="bold">netflix</span>roulette
                </span>
            </div>
            <div className="delimiter-2" />
        </div>
    );
};
