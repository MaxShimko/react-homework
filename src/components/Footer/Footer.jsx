import React from 'react';

import './style.scss';

export default function Footer() {
    return (
        <div className="footer">
            <div className="delimiter-2" />
            <div>
                <span className="logotype">
                    <span className="bold">netflix</span>roulette
                </span>
            </div>
            <div className="delimiter-2" />
        </div>
    );
};
