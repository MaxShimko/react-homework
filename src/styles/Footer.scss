.footer {
  background: #555555;
  text-align: center;
}
