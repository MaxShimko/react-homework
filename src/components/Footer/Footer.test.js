import React from 'react';
import { configure, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Footer from './Footer';
import {describe, expect, it} from '@jest/globals';

configure({ adapter: new Adapter() });

describe('<Footer />', () => {
    it('renders without crashing', () => {
        const component = render(<Footer />);
        expect(component).toMatchSnapshot();
    });
});
