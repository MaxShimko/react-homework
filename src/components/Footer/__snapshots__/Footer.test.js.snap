// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders without crashing 1`] = `
<div
  className="footer"
>
  <div
    className="delimiter-2"
  />
  <div>
    <span
      className="logotype"
    >
      <span
        className="bold"
      >
        netflix
      </span>
      roulette
    </span>
  </div>
  <div
    className="delimiter-2"
  />
</div>
`;
